iocore/net/P_SNIActionPerformer.h

```diff
@@ -67,8 +67,9 @@ class DisableH2 : public ActionItem
     auto ssl_vc     = reinterpret_cast<SSLNetVConnection *>(cont);
     auto accept_obj = ssl_vc ? ssl_vc->accept_object : nullptr;
     if (accept_obj && accept_obj->snpa && ssl_vc) {
-      auto nps = snpsMap.get(accept_obj->id);
-      ssl_vc->registerNextProtocolSet(reinterpret_cast<SSLNextProtocolSet *>(nps));
+      if (auto it = snpsMap.find(accept_obj->id); it != snpsMap.end()) {
```

Comment:
Here's this bloody pattern again... You make it look like a foor-loop, it's not an iterator!!

```
if (auto obj = snpsMap.find(accept_obj->id); obj != snpsMap.end()) {
```